### ISSUE TITLE
triggering mediascanner 

### ISSUE THREAD

**Issue by shivamguptaagatsa** [master-atul/react-native-simple-download-manager#3]
hey there, is it possible to trigger mediascanner from this module? because when i download an image it did not show in gallery. I have to restart my phone to trigger the mediascanner.

**Comment by master-atul** [comment]
Yes, this should be possible. Will try to implement this, but will take some time.  Meantime if you have some time .. could you raise a PR for the same ?
